content/blog/2020-04-06-april-20-dvc-heartbeat.md

```diff
@@ -14,10 +14,11 @@ descriptionLong: |
   projects by our users and big ideas about best practices in ML and data
   science.
 picture: 2020-04-06/april_header.png
+# prettier-ignore
 pictureComment:
-  A view from [Barrancas del
-  Cobre](https://en.wikipedia.org/wiki/Copper_Canyon), shot by Jorge Orpinel
-  Pérez. Jorge has mastered the art of working on DVC remotely.
+  A view from
+  [Barrancas del Cobre](https://en.wikipedia.org/wiki/Copper_Canyon), shot by
```

Comment:
BTW funny that we need to prettier-ignore inside frontmatter so that Pretty doesn't break the same format it enforces in regular md haha. I guess it has no way to know that this string in YAML is supposed to be md.

Reply:
hmm ... that's strange - I though that prettier doesn't touch frontmatter (it def doesn't wrap lines for example)

Reply:
It's wrapping them (no changes to config), and since it doesn't know it's md, it will break the link.

Reply:
I mean "break" it into 2 lines (the link still works I think).

Reply:
okay, try to use `pictureComment: |` (Block Style Indicator)

Reply:
🤦 yeah I didn't think about that for some reason. That works 👍